content/blogs/control-vmware-with-kestra.md

```diff
@@ -127,13 +128,17 @@ Until now, VMware automation often required custom scripts, separate tools, or m
 - **Event-Driven**: React to infrastructure changes as they happen.
 - **Scalable**: Tie VM operations into CI/CD, testing, and business process automation.
 
----
+## A Smarter Way to Orchestrate VMware
+
+Virtual machines are still the foundation of enterprise infrastructure—but managing them shouldn’t be stuck in the past. The VMware plugin for Kestra EE brings first-class orchestration to your virtualization stack, making it easy to:
 
-## Conclusion
+- Automate repetitive VM operations like provisioning, powering on/off, and cloning
+- Coordinate lifecycle management across environments in a version-controlled, declarative way
+- React to real-time changes and trigger downstream workflows without manual intervention
 
-Virtual machines remain the backbone of many enterprise systems, and now, with Kestra’s new VMware plugin, orchestrating them is easier, faster, and more robust than ever. Whether you’re scaling environments with templates, managing snapshots for safety, or reacting to real-time VM events, Kestra brings orchestration simplicity to your VMware estate.
+Infrastructure automation is about: speed, resilience, traceability, and making your platform team more effective. This plugin helps get you there.
```

Comment:
"This plugin helps you to get there" ?

Reply:
Fixed: https://github.com/kestra-io/docs/pull/2865/commits/0f08953b5e3492d2a32d165891d3105d9f059427